ckan/lib/dictization/model_dictize.py

```diff
@@ -631,7 +631,7 @@ def activity_dictize(activity: model.Activity, context: Context,
         # the dataset/group/org/custom obj. we need the title to display it
         # in the activity stream.
         activity_dict['data'] = {
```

Comment:
@amercader  Yes, i agree that we can remove `include_data` flag and return the whole activity object.It could be usefull not only in the snippets but also in helpers if we need to compare some other field from previous activities